ladybug_comfort/__init__.py

```diff
@@ -8,7 +8,7 @@
         colum refers to a degree of altitude.
 """
 
-__version__ = '0.2.0'
+__version__ = '0.1.1'
```

Comment:
is this the correct version?

Reply:
I don't think so. I am going to amend the previous commit and fix it since I realized some changes I made to ladybug-core affected the tests here.

Reply:
This is fixed.